Code/Editor/EditorPreferencesPageGeneral.cpp

```diff
@@ -92,6 +97,14 @@ void CEditorPreferencesPage_General::Reflect(AZ::SerializeContext& serialize)
             ->DataElement(AZ::Edit::UIHandlers::CheckBox, &GeneralSettings::m_restoreViewportCamera, EditorPreferencesGeneralRestoreViewportCameraSettingName, "Keep the original editor viewport transform when exiting game mode.")
             ->DataElement(AZ::Edit::UIHandlers::CheckBox, &GeneralSettings::m_enableSceneInspector, "Enable Scene Inspector (EXPERIMENTAL)", "Enable the option to inspect the internal data loaded from scene files like .fbx. This is an experimental feature. Restart the Scene Settings if the option is not visible under the Help menu.");
 
+        editContext->Class<GlobalSaveSettings>("Global Save Settings (File>Save & Ctrl+S)", "")
+            ->DataElement(
+                AZ::Edit::UIHandlers::ComboBox, &GlobalSaveSettings::m_saveAllPrefabsPreference, "Save Prefabs Preference",
```

Comment:
should this be called like: "Save Prefabs inside Level" ?

Reply:
I've intentionally avoided using the word 'level' in the header so that this stays the same in the future if levels become optional and users can start with a scene and a prefab.

Regarding Ctrl+S, this only shows up if the user hovers on the header. And I think being explicit on where this change affects currently is better than excluding it. If the behavior of ctrl+s changes later, this can be excluded but I doubt that it will change anytime soon.

Reply:
The problem is that Save can be overriden to not be Ctrl+S, for example CTRL+O or whatever, hence Ctrl+S won't make sense here

Reply:
Yes, this is for saving levels specifically. Changed the name to 'Level Save Settings' to make things clearand removed 'Ctrl+S' from the hint.